File: Pattern48.java
import java.util.Scanner;
public class Pattern48{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter the number of rows : ");
        int rows = sc.nextInt();

        int count = 1;
        for(int i = 1; i <= rows; i++){
            for(int j = 1; j <= i; j++){
                System.out.print(count + " ");
                count++;
            }
            System.out.println();
        }
    }
}

File: PATTERNS/Pattern48/Pattern48.py
n = int(input("Enter the number of rows: "))
num = 0

    
for i in range(1, n+1):
    if i % 2 == 0:
        num += i
        for j in range(1, i+1):
            print(num, end=" ")
            num -= 1
        num += i
    else:
        for j in range(1, i+1):
            num += 1
            print(num, end=" ")
            
    print()
